### PR TITLE
Symfony insight fixes

### PR DESCRIPTION
**Pull request by craigh**
fixes for symfony insight concerns

### CODE REVIEW

File: src/Kernel.php [hunk 1]
```diff
@@ -28,7 +28,9 @@ class Kernel extends ZikulaKernel
 
     public function registerBundles(): iterable
     {
-        $bundleHelper = new PersistedBundleHelper($_ENV['DATABASE_URL'] ?? '');
+        $databaseUrl = $this->booted && $this->getContainer()->hasParameter('database_url') ? $this->getContainer()->getParameter('database_url') : '';
+        $bundleHelper = new PersistedBundleHelper($databaseUrl);
+//        $bundleHelper = new PersistedBundleHelper($_ENV['DATABASE_URL'] ?? '');
         $bundles = require $this->getProjectDir() . '/config/bundles.php';
```
Comment (craigh) [inline src/Kernel.php:34]: @Guite this is an idea but I've not really tested it. it requires the parameter change in services.yaml too. I'm not sure its better practice then using the GLOBAL. :-) wonder what you think.

Comment (craigh) [reply src/Kernel.php:34]: actually, I'm not even sure it WORKS. I'll be interested to hear your thoughts.

Comment (Guite) [reply src/Kernel.php:34]: Yes. Using a parameter seems the way to go :+1:

File: src/system/ExtensionsModule/Twig/Extension/ExtensionsExtension.php [hunk 2]
```diff
@@ -19,16 +19,6 @@
 
 class ExtensionsExtension extends AbstractExtension
 {
-    /**
-     * @var TranslatorInterface
-     */
-    private $translator;
-
-    public function __construct(TranslatorInterface $translator)
```
Comment (Guite) [inline src/system/ExtensionsModule/Twig/Extension/ExtensionsExtension.php:27]: Please remove `use Symfony\Contracts\Translation\TranslatorInterface;` too (see StyleCI).

File: src/Kernel.php (hunk 1 above)
Comment (craigh) [reply src/Kernel.php]: doesn't work.

**Pull request merged**